application/helpers/update/updatedb_helper.php

```diff
@@ -2264,6 +2276,68 @@ function db_upgrade_all($iOldDBVersion, $bSilent = false)
     Yii::app()->setConfig('Updating', false);
     return true;
 }
+
+
+/**
+ * @param string $sMySQLCollation
```

Comment:
Can you look at code if here : it's currentColation, with mssql+pgsql or mysql.

Applys same collation than https://github.com/LimeSurvey/LimeSurvey/pull/1019/files#diff-4a95ed4f80f8713aaa18250f998ddfafL138